packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.test.tsx

```diff
@@ -66,22 +69,19 @@ const initialQueryRequest: QueryBundleRequest = {
 }
 
 describe('QueryWrapper', () => {
+  beforeAll(() => server.listen())
```

Comment:
Update tests to use MSW. Since our MSW handler utilizes the partMask, these changes catch the regression.